de_workloads/processing/gold_movies_example/README.md

```diff
@@ -0,0 +1,9 @@
+# Data Processing Template Workload
```

Comment:
Might be good to update this readme with a brief overview of the workload (and we should do the same for the others).